packages/p2p/src/components/buy-sell/no-ads/no-ads.jsx

```diff
@@ -26,7 +33,9 @@ const NoAds = () => {
                         size='s'
                         weight='bold'
                     >
-                        <Localize i18n_default_text='No ads for this currency 😞' />
+                        <Localize
+                            i18n_default_text={is_ads_page ? 'You have no ads.' : 'No ads for this currency 😞'}
```

Comment:
So just FYI you can't use ternaries wihin the localize as this will cause issues with translations, so make sure to separate it 👍 

```suggestion

     { is_ads_page ? 
               <Localize  i18n_default_text='You have no ads.'  /> 
               : <Localize  i18n_default_text='No ads for this currency 😞'  />
     }
```